tools/scripts/pub.mjs

```diff
@@ -101,8 +100,23 @@ await question('Are you sure you want to publish to? (y/n)', {
     // await 1 second
     await new Promise((resolve) => setTimeout(resolve, 1000));
 
+    // first publish the wrapped keys lit actions by running yarn publish-lit-actions in ./packages/wrapped-keys-lit-actions
+    await spawnCommand(
```

Comment:
_Any_ time we publish the wrapped-keys-lit-actions package, even if done manually using a direct NPM command, we want this to happen -- can we make this happen as part of a pre-publish NPM script instead of bespoke logic and shell execution here?

e.g. -- if we add this to `prepublish` it should always run when publishing, as part of that process...

https://docs.npmjs.com/cli/v10/using-npm/scripts#life-cycle-scripts

Reply:
Good point. Changed this to this two things
- on LAWK: a prepublishOnly script to build and publish LAs on IPFS when publishing
- bonus, on LA: a prepare script to update its CIDs on publish but also on install. Installation should be trivial but useful to allow us to spot any issue